tests/std/tests/P0660R10_jthread_and_cv_any/test.cpp

```diff
@@ -0,0 +1,243 @@
+        struct overload_detector {
+            bool* p_called;
+            void operator()(stop_token, int i) const {
+                assert(i == 1729);
+                *p_called = true;
+            }
+            void operator()(int) const {
+                assert(false);
+            }
+        };
+
+        {
+            jthread worker{overload_detector{&called}, 1729};
+            (void) worker;
+        }
+
+        assert(called);
+    }
+
+    { // move ctor
+        jthread worker{[] {}};
+        auto worker_source = worker.get_stop_source();
+        {
+            jthread moved{move(worker)};
+            assert(moved.get_stop_source() == worker_source);
+            assert(moved.joinable());
+            assert(worker.get_stop_source() != worker_source);
+            assert(!worker.joinable());
+        }
+    }
+
+    { // move assign
+        jthread worker_a{[] {}};
+        auto source_a = worker_a.get_stop_source();
+        jthread worker_b{[] {}};
+        auto id_b     = worker_b.get_id();
+        auto source_b = worker_b.get_stop_source();
+        worker_a      = move(worker_b);
+        assert(source_a.stop_requested());
+        assert(id_b == worker_a.get_id());
+        assert(!source_b.stop_requested());
+        assert(worker_a.get_stop_source() == source_b);
+        assert(!worker_b.joinable());
+    }
+
+    { // self move assign, currently specified to try to cancel and join
+        jthread worker{[] {}};
+        auto source = worker.get_stop_source();
+        worker      = move(worker);
+        assert(!worker.joinable());
+        assert(source.stop_requested());
+    }
+
+    { // swaps
+        jthread worker_a{[] {}};
+        auto id_a     = worker_a.get_id();
+        auto source_a = worker_a.get_stop_source();
+        auto token_a  = worker_a.get_stop_token();
+        jthread worker_b{[] {}};
+        auto id_b     = worker_b.get_id();
+        auto source_b = worker_b.get_stop_source();
+        auto token_b  = worker_b.get_stop_token();
+
+        assert(id_a != id_b);
+        assert(source_a != source_b);
+        assert(token_a != token_b);
+
+        worker_a.swap(worker_b);
+        assert(worker_a.get_id() == id_b);
+        assert(worker_a.get_stop_source() == source_b);
+        assert(worker_b.get_id() == id_a);
+        assert(worker_b.get_stop_source() == source_a);
+        swap(worker_a, worker_b);
+        assert(worker_a.get_id() == id_a);
+        assert(worker_a.get_stop_source() == source_a);
+        assert(worker_b.get_id() == id_b);
+        assert(worker_b.get_stop_source() == source_b);
+    }
+
+    { // join
+        jthread worker{[] {}};
+        auto source = worker.get_stop_source();
+        worker.join();
+        assert(!worker.joinable());
+        assert(worker.get_stop_source() == source);
+        assert(!source.stop_requested());
+        assert(source.stop_possible());
+    }
+
+    // TRANSITION, MSFT-11107628 "_Exit allows cleanup in other DLLs"

```

Comment:
In the libcxx skip lists we cited this as OS-11107628 but it no longer loads (and I don't know where it moved to).

Reply:
Yeah I'm not sure exactly where it went but we should cite the same way.